docs/blog/2018-10-06-deploying-gatsby-to-azure/index.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Deploying a Gatsby blog to Azure"
+author: Doug McDonald
+date: 2018-10-06
+image: images/rocket.jpg
+showImageInArticle: false
+tags: ["deployment", "azure"]
+---
+
+In this post we will walk through the process of deploying a Gatsby blog to Microsoft Azure Storage using VS Code.
+
+## Prerequisites
+
+To complete this tutorial you will need the following tools:
+
+- Node - To run the rest of the tools
+- Gatsby CLI - To create your blazingly fast blog!
+- VS Code - as a code editor
+
+### Prerequisites Installation
+
+Install Node from the [Node website](https://nodejs.org/en/) and check it's installed by checking the version from your terminal.
+
+```bash
+$ node -v
+$ v10.9.0
+```
+
+Install VS Code from the [VS Code website](https://code.visualstudio.com/).
+
+Next, we can install the important part! [Gatsby](https://gatsbyjs.org/) from your terminal, run:
+
+```bash
+$ npm install --global gatsby-cli
++ gatsby-cli@2.4.2
+updated 1 package in 4.868s
+```
+
+## Creating your blog
+
+Next you will create the blog from the handy [Gatsby Starter Blog](https://github.com/gatsbyjs/gatsby-starter-blog).
+
+> If you already have a site you want to deploy, skip forwards to [Setting up Azure](#setting-up-azure)
+
+From your terminal run
+
+```bash
+$ gatsby new azure-blog https://github.com/gatsbyjs/gatsby-starter-blog
+
+info Creating new site from git: https://github.com/gatsbyjs/gatsby-starter-blog.git
+Cloning into 'azure-blog'...
+remote: Enumerating objects: 1, done.
+remote: Counting objects: 100% (1/1), done.
+remote: Total 1139 (delta 0), reused 0 (delta 0), pack-reused 1138
+Receiving objects: 100% (1139/1139), 2.14 MiB | 2.95 MiB/s, done.
+Resolving deltas: 100% (636/636), done.
+success Created starter directory layout
+info Installing packages...
+...
+```
+
+You can now view your website by running
+
+```bash
+$ cd azure-blog
+$ gatsby develop
+```
+
+You should now see your blog in the flesh!
+
+![Gatsby blog starter](blogstarter.png)
+
+## Setting up Azure
+
+Microsoft Azure is a cloud hosting provider which offers a wide range of tools and services. Today you are going to use Azure Storage to host our static Gatsby blog.
+
+> If you don't have an Azure account, [sign up today](https://azure.microsoft.com/en-us/free/) for a free 30 day account with $200 in Azure credits.
```

Comment:
Might be worth calling out that you _must_ provide a credit card to sign up. That could save people some frustration if they're unwilling to add one.

Reply:
I'll add something to that effect, interestingly it's always a bit different writing from the perspective of having pre-existing accounts / logins. Thanks for the feedback